examples/hotrod/cmd/root.go

```diff
@@ -51,23 +58,38 @@ func Execute() {
 
 func init() {
 	RootCmd.PersistentFlags().StringVarP(&metricsBackend, "metrics", "m", "expvar", "Metrics backend (expvar|prometheus)")
-	RootCmd.PersistentFlags().StringVarP(&jAgentHostPort, "jaeger-agent.host-port", "a", "0.0.0.0:6831", "String representing jaeger-agent host:port")
+	RootCmd.PersistentFlags().StringVarP(&jAgentHostPort, "jaeger-agent.host-port", "a", "0.0.0.0:6831", "String representing jaeger-agent UDP host:port")
```

Comment:
should this also say that http host:port is accepted as well?

Reply:
we could say that, my main hesitation was that the property is called `jaeger-agent.host-port`, whereas an HTTP endpoint would be running in the collector, not the agent.

Reply:
doh, in that case, should we just add a new flag for jaeger-client.reporting-host-port? and say the old one is deprecated? it is a demo app, not sure if many would care.

Reply:
for now I added http to the description. I would rather not create another flag, as I think there's a better way - #920